common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -38,10 +97,876 @@ innovation_taming_the_skies = {
 
 	potential = {
 		is_giant_culture_trigger = no
+		NOT = {
+			has_aerial_maa_trigger = yes
+		}
 	}
 
 	unlock_maa = aerial_unit
 
 	flag = global_regional
 	flag = tribal_era_regional
 }
+
+### MAA TECHS
+
+####Tribal
+innovation_archmage = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @archmage
+
+	potential = {
+		AND = {
+			OR = {
+				this.culture_group = culture_group:human_group
+				this = culture:blood_elf
+				this = culture:high_elf
+			}
+			NOT = {
+				this = culture:pirate_culture
+				this = culture:wastewander	
+			}
+		}
+	}
+
+	unlock_maa = archmage
+
+	flag = global_maa
+}
+
+
+innovation_blademaster = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @horde_special
+
+	potential = {
+		this = culture:burning_blade
+	}
+
+	unlock_maa = blademaster
+
+	flag = global_maa
+}
+
+innovation_buccaneer = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @buccaneer
+
+	potential = {
+		OR = {
+			this = culture:pirate_culture
+			this = culture:blackwater
+		}
+	}
+
+	unlock_maa = buccaneer
+
+	flag = global_maa
+}
+
+innovation_brute = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @brute
+
+	potential = {
+		this.culture_group = culture_group:gnoll_group
+	}
+
+	unlock_maa = brute
+
+	flag = global_maa
+}
+
+innovation_candleholder = {
+	group = culture_group_regional
+	culture_era = culture_era_tribal
+	icon = @candleholder
+
+	potential = {
+		this.culture_group = culture_group:kobold_group
+	}
+
+	unlock_maa = candleholder
+
+	flag = global_maa
+}
+
+innovation_dervishes = {
```

Comment:
Reminding there is oasis terrain, it's desert too basically.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/133156998-565f5b5f-8cef-44c4-aac2-66d8fb15cc69.png)

</details>

Don't get why hills get a bonus, but it may be.

Reply:
👌 will add